indexers/staking-squid/src/events/bundle.ts

```diff
@@ -9,20 +15,105 @@ export function processBundleStoredEvent(
+  const operatorId = Number(bundleAuthor);
+
+  const sealedHeader = extrinsic.call?.args.opaqueBundle
+    .sealedHeader as SealedBundleHeader;
+  const domain = getOrCreateDomain(cache, block, domainIdNum);
   const operator = getOrCreateOperator(cache, block, operatorId);
+  const account = getOrCreateAccount(cache, block, operator.accountId);
+
+  const receipt = sealedHeader.header.receipt as ExecutionReceipt;
+  const { transfers } = receipt;
+
+  const totalTransfersIn = transfers.transfersIn.reduce(
+    (acc, [, amount]) => acc + BigInt(amount),
+    BigInt(0)
+  );
+  const transfersInCount = transfers.transfersIn.length;
+
+  const totalTransfersOut = transfers.transfersOut.reduce(
+    (acc, [, amount]) => acc + BigInt(amount),
+    BigInt(0)
+  );
+  const transfersOutCount = transfers.transfersOut.length;
+
+  const totalRejectedTransfersClaimed =
+    transfers.rejectedTransfersClaimed.reduce(
+      (acc, [, amount]) => acc + BigInt(amount),
+      BigInt(0)
+    );
+  const rejectedTransfersClaimedCount =
+    transfers.rejectedTransfersClaimed.length;
+
+  const totalTransfersRejected = transfers.transfersRejected.reduce(
+    (acc, [, amount]) => acc + BigInt(amount),
+    BigInt(0)
+  );
+  const transfersRejectedCount = transfers.transfersRejected.length;
+
+  const totalVolume = totalTransfersIn + totalTransfersOut;
+
+  const {
+    domainBlockNumber,
+    consensusBlockNumber,
+    consensusBlockHash,
+    blockFees,
+  } = receipt;
+
+  const bundle = createBundle(account.id, domain.id, operator.id, {
+    domainBlockNumber: Number(domainBlockNumber),
+    consensusBlockNumber: Number(consensusBlockNumber),
+    consensusBlockHash,
+    totalTransfersIn,
+    transfersInCount,
+    totalTransfersOut,
+    transfersOutCount,
+    totalRejectedTransfersClaimed,
+    rejectedTransfersClaimedCount,
+    totalTransfersRejected,
+    transfersRejectedCount,
+    totalVolume,
+    consensusStorageFee: BigInt(blockFees.consensusStorageFee),
+    domainExecutionFee: BigInt(blockFees.domainExecutionFee),
+    burnedBalance: BigInt(blockFees.burnedBalance),
+  });
+
+  cache.bundles.set(bundle.id, bundle);
 
-  domain.lastDomainBlockNumber = lastDomainBlockNumber;
+  domain.lastDomainBlockNumber = Number(domainBlockNumber);
+  domain.totalTransfersIn += totalTransfersIn;
+  domain.transfersInCount += transfersInCount;
+  domain.totalTransfersOut += totalTransfersOut;
+  domain.transfersOutCount += transfersOutCount;
+  domain.totalRejectedTransfersClaimed += totalRejectedTransfersClaimed;
+  domain.rejectedTransfersClaimedCount += rejectedTransfersClaimedCount;
+  domain.totalTransfersRejected += totalTransfersRejected;
+  domain.transfersRejectedCount += transfersRejectedCount;
+  domain.totalVolume += totalVolume;
+  domain.totalConsensusStorageFee += BigInt(blockFees.consensusStorageFee);
+  domain.totalDomainExecutionFee += BigInt(blockFees.domainExecutionFee);
+  domain.totalBurnedBalance += BigInt(blockFees.burnedBalance);
```

Comment:
I think this might lead to duplicating these values at the domain level, as there can be many bundles with the same execution receipt in a given block. Execution fees are shared among all operators who produce bundles with execution receipts extending the last domain block. @vedhavyas can you confirm if this is true?

Reply:
As you mentioned @jfrank-summit the bundle store event seems to be emitted for each operators so indeed I can't just increment at each event 🫠 

But why the same operators have multiple bundle store event for the same bundle?

![image](https://github.com/user-attachments/assets/a64a1510-2b97-4379-b738-811b883fc433)

Reply:
> But why the same operators have multiple bundle store event for the same bundle?

@vedhavyas @NingLin-P do you have an answer?

Reply:
>Execution fees are shared among all operators who produce bundles with execution receipts extending the last domain block.

Yeah, this is true

>But why the same operators have multiple bundle store event for the same bundle?

Not the same bundle, the operator can produce multiple bundles and included in the same consensus block, each bundle will emit an event. Also, we do have a check to reject duplicated bundles in the same block and the previous block.

Reply:
I partially addressed the issue in https://github.com/autonomys/astral/pull/796
(so incrementing the domain only on new bundle)
I'm still unsure what to do with the operator part

Reply:
Which operator part?

Reply:
A few lines below your comment, I do the same on the operator object...